packages/web/src/components/track/PremiumConditionsPill.module.css

```diff
@@ -36,3 +6,7 @@
 .spinner g path {
   stroke: var(--static-white);
 }
+
+.button {
+  height: var(--unit-6);
```

Comment:
flagging as a button override that I should follow up with design about

Reply:
thank yo ufor making it a button :) should it be a radio button?